### ISSUE TITLE
Error: 'AnimatedScale' is imported from both

### ISSUE THREAD

**Issue by shalom-aviv** [Idean/Flutter-Neumorphic#227]
Have next error when run build for ios
```
/flutter_neumorphic-3.1.0/lib/src/widget/switch.dart:167:20: 
Error: 
  'AnimatedScale' is imported 
from both 'package:flutter/src/widgets/implicit_animations.dart'  and
 'package:flutter_neumorphic/src/widget/animation/animated_scale.dart'.
```

```
Flutter 2.4.0-4.0.pre • channel dev • https://github.com/flutter/flutter.git
Framework • revision cc00e7e6bc (4 days ago) • 2021-07-11 18:21:02 -0400
Engine • revision ed25e8f01e
Tools • Dart 2.14.0 (build 2.14.0-301.0.dev)
```

**Comment by diarmuidmcg** [comment]
Just started getting this error as well. Not sure how to fix.

**Comment by enigma-rdev** [comment]
Change all the instances of AnimatedScale in this package to something else and it will work

**Comment by lyntree** [comment]
Hi,

As much as it sounds that simple, I think you might want to run some tests on this package against the Flutter Master & Stable channel.  I hadn't touched my app since early July, it was working perfectly with this UI.. 

Then yesterday happened .. Everything including my building tools has broken on me..

The weirdest thing is, I'm not even using an 'animated scale' let alone importing it...  so why is it suddenly flagging up in our code?

Can you please look into this, otherwise I might have to rip piece by piece of this UI out and that would be a shame :(

**Comment by lyntree** [comment]
Just to Add a Note on this,  I even tried removing the imports from those files and cleaning the build that did not work.
I was sitting on the 'Master Channel', then I switched to 'Stable' and 'Upgraded' and magically, the error has disappeared...

I would recommend that if your team could please look into this as maybe when flutter releases their next 'package', this is an early warning that it will break completly. Hope this helps!

**Comment by enigma-rdev** [comment]
hahah magically,
Let me see if I can get someone to fix this. Thanks

**Comment by cs-milan-sadariya** [comment]
> Just to Add a Note on this, I even tried removing the imports from those files and cleaning the build that did not work.
> I was sitting on the 'Master Channel', then I switched to 'Stable' and 'Upgraded' and magically, the error has disappeared...
> 
> I would recommend that if your team could please look into this as maybe when flutter releases their next 'package', this is an early warning that it will break completly. Hope this helps!

It's worked for me. Thanks

**Comment by roked** [comment]
> Just to Add a Note on this, I even tried removing the imports from those files and cleaning the build that did not work.
> I was sitting on the 'Master Channel', then I switched to 'Stable' and 'Upgraded' and magically, the error has disappeared...
> 
> I would recommend that if your team could please look into this as maybe when flutter releases their next 'package', this is an early warning that it will break completly. Hope this helps!

Easy fix!! 

```
flutter channel stable
flutter upgrade
```

**Comment by lyntree** [comment]
Just coming back to this, I've had to switch back to the master channel of Flutter.. unfortunately, the error does pop up again, is there a way we can get this fixed on master and how long could it take?

**Comment by Raphael-Jose-Ferraro-IRForever** [comment]
Same problem, waiting for fix.

**Comment by lyntree** [comment]
Do we know when this might happen for a fix, otherwise I might have to pull this entire package out of my app, which I won't be looking forward to doing :(

**Comment by seunghwanly** [comment]
The same issue with the `AnimatedScale` widget, is it only fine using in the `stable` channel? 🥲

**Comment by lyntree** [comment]
Realistically, I don't think this team will release another fix for this, because 'technically' it's already gone through master, dev, beta down to stable so, you either have to sit on Master until any issues are fixed on master passed back down onto Stable to get the fix for this.

**Comment by LeonBraeunig** [comment]
seems like that flutter 2.5 does not fixed that

**Comment by lyntree** [comment]
It's already been fixed, its on stable channel

**Comment by juanpare-peya** [comment]
Meanwhile you can use the package like this:

```yaml
  flutter_neumorphic:
    git:
      url: "https://github.com/alionour/Flutter-Neumorphic"
```

**Comment by LeonBraeunig** [comment]
> Meanwhile you can use the package like this:
> 
> ```yaml
>   flutter_neumorphic:
>     git:
>       url: "https://github.com/alionour/Flutter-Neumorphic"
> ```

worked!! thanks a lot. :)

**Comment by miguelpruivo** [comment]
Fixed with 3.1.1. 🎉
